components/ome-jxr/src/ome/jxr/JXRException.java

```diff
@@ -28,7 +28,9 @@
 /**
  * JXRException is thrown when an error is encountered during the broadly
  * understood parsing of a JPEG XR image file.
+ * @deprecated See <a href="http://lists.openmicroscopy.org.uk/pipermail/ome-users/2015-September/005616.html">mailing-list thread</a> and <a href="http://blog.openmicroscopy.org/file-formats/community/2016/01/06/format-support">blog post</a>
```

Comment:
Seconding referencing the blog post alone, since the mailing list thread primarily references .mrxs support when most people will be looking for this in the context of .czi support.